molecule/test-minikube/converge.yml

```diff
@@ -73,7 +73,7 @@
             kind: Namespace
             api_version: v1
 
-        - name: Create the tower.ansible.com/v1alpha1.Tower
+        - name: Create the tower.ansible.com Tower Resource
```

Comment:
For consistency, we should have this line and the one in `test-local/converge.yml` be the same. It was using the CRD-based-path, but we can just call it 'Create the Tower Custom Resource' for more readability and to take the version out of this string (easier maintenance).